src/java/org/jivesoftware/openfire/pubsub/PubSubServiceInfo.java

```diff
@@ -1,19 +1,47 @@
 package org.jivesoftware.openfire.pubsub;
 
+import org.jivesoftware.openfire.XMPPServer;
```

Comment:
Something is off with the indentation in this file. Could you make it consistent please?

Reply:
Done, also re formatted pubsub-form-table.jsp